ext/console/01_console.js

```diff
@@ -1371,6 +1382,7 @@ function getKeys(value, showHidden) {
     try {
       keys = ObjectKeys(value);
     } catch (err) {
+      console.log(err);
```

Comment:
Is this intentional?

Reply:
I'm still debugging the mentioned regression

Reply:
Oh, sorry I didn't realize it wasn't ready. Yeah, draft PR is better :)

Reply:
Any ideas about the mentioned regression ?